mllib/src/main/scala/org/apache/spark/ml/tuning/ValidatorParams.scala

```diff
@@ -76,6 +77,18 @@ private[ml] trait ValidatorParams extends HasSeed with Params {
     }
     est.copy(firstEstimatorParamMap).transformSchema(schema)
   }
+
+  /**
+   * Instrumentation logging for tuning params including the inner estimator and evaluator info.
+   *
+   * @param instrumentation instrumentation logger
+   */
+  protected def logTuningParams(instrumentation: Instrumentation[_]): Unit = {
+    instrumentation.log(compact(render(map2jvalue(Map[String, JValue](
```

Comment:
I'd say just use instrumentation.logNamedValue for each of these, rather than handling JSON here.